guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc

```diff
@@ -18,7 +18,8 @@ Installing GSS-proxy and {nfs-client-package}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-install} sssd adcli realmd ipa-python-compat krb5-workstation samba-common-tools
+# {project-package-install} adcli ipa-python-compat krb5-workstation \
+oddjob oddjob-mkhomedir realmd samba-common-tools sssd
```

Comment:
I've never had to specify `oddjob*` to get the setup working

Reply:
So this is what the `realm join` command actually does when I don't install `oddjob*` before running it:
```
 * Couldn't find file: /usr/sbin/oddjobd
 * Required files: /usr/sbin/oddjobd, /usr/libexec/oddjob/mkhomedir, /usr/sbin/sssd, /usr/bin/net
 * Resolving required packages
 * Installing necessary packages: oddjob oddjob-mkhomedir samba-common-tools
```
Can't hurt to pre-install it manually, not relying on the above to work. Also, it's the documented way. So yes, let's keep it.